R/aaa-auto.R

```diff
@@ -1370,6 +1370,27 @@ feedback_arc_set_impl <- function(graph, weights=NULL, algo=c("approx_eades", "e
   res
 }
 
+feedback_vertex_set_impl <- function(graph, weights=NULL, algo=EXACT_IP) {
```

Comment:
@Antonov548 Again, `algo=EXACT_IP` is invalid code, which needs to be dealt with, either by setting up translation for the new type, or removing code generation for this function temporarily. Exposing this function should be really easy, as it's near-identical to the feedback arc set one (so it's a copy and paste job). Ref: https://github.com/igraph/rigraph/issues/1447